### PR TITLE
Load mvs keep camera

### PR DESCRIPTION
**Pull request by midlik**
# Description
I added `keepCamera` option to `loadMVS` function in MolViewSpec extension. 
I also slightly changed the way `radiusMax` is computed, to avoid aggressive clipping in cases when the camera target is not aligned with the scene's bounding sphere.

## Actions

- [x] Added description of changes to the `[Unreleased]` section of `CHANGELOG.md`
- [x] Updated headers of modified files
- [x] Added my name to `package.json`'s `contributors`
- [ ] (Optional but encouraged) Improved documentation in `docs`

### CODE REVIEW

**Comment by midlik** [comment]
@arose could you please check the change I made in `[src/mol-canvas3d/canvas3d.ts](https://github.com/molstar/molstar/pull/1101/files#diff-a387ffcb1f053ad2d4ad3f869712109e398311a2846d9eb2a5db831956356adf)`?

@dsehnal was not sure if it can break anything.

File: src/mol-canvas3d/canvas3d.ts [hunk 1]
```diff
@@ -602,7 +602,8 @@ namespace Canvas3D {
                 const focus = camera.getFocus(center, radius);
                 const next = typeof nextCameraResetSnapshot === 'function' ? nextCameraResetSnapshot(scene, camera) : nextCameraResetSnapshot;
                 const snapshot = next ? { ...focus, ...next } : focus;
-                camera.setState({ ...snapshot, radiusMax: getSceneRadius() }, duration);
+                const offset = snapshot.target ? Vec3.distance(snapshot.target, scene.boundingSphere.center) : 0;
```
Comment (arose) [inline src/mol-canvas3d/canvas3d.ts:605]: Hmm, could you set `sceneRadiusFactor` instead? This definitely breaks things.

Comment (dsehnal) [reply src/mol-canvas3d/canvas3d.ts]: Thanks, that makes more sense.

**Review by dsehnal — APPROVED** [review]

File: src/mol-canvas3d/canvas3d.ts (hunk 1 above)
Comment (midlik) [reply src/mol-canvas3d/canvas3d.ts]: Do you mean like this?: https://github.com/molstar/molstar/pull/1101/commits/00ad11dd6b170ad5815c12588dbd173aca47b01d

A disadvantage of tweaking `sceneRadiusFactor` is that it then stays changed forever (`radiusMax` in the original solution would go back to normal on the next camera reset).

Comment (arose) [reply src/mol-canvas3d/canvas3d.ts]: I suppose if you don't want the aggressive clipping, you want it forever?

Also note that you have access to the camera object and could change things there from your extension.

Comment (dsehnal) [reply src/mol-canvas3d/canvas3d.ts]: The issue this is trying to solve it this:

image

Without either allowing bigger `radiusMax` or increasing the factor it seems quite hard to fix this, where the use case is loading a different state without moving the camera.

Any other ideas how to solve this @arose?

Comment (arose) [reply src/mol-canvas3d/canvas3d.ts]: > Any other ideas how to solve this @arose?

nothing at the moment

**Review by arose — APPROVED** [review]

**Pull request merged**